Bottom: 5b702dac7485a3a660ef8c2611b83386f626b571
Top:    2f4cd40057e443e80cf5eae99eab5efd48410c04
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-28 01:11:51 +0100

HIDs: Remove NOOP set_draw_faded implementations


---

diff --git a/src/hid/batch/batch.c b/src/hid/batch/batch.c
index a32e89b..3ccca3a 100644
--- a/src/hid/batch/batch.c
+++ b/src/hid/batch/batch.c
@@ -208,11 +208,6 @@ batch_set_draw_xor (hidGC gc, int xor_set)
 }
 
 static void
-batch_set_draw_faded (hidGC gc, int faded)
-{
-}
-
-static void
 batch_draw_line (hidGC gc, int x1, int y1, int x2, int y2)
 {
 }
@@ -359,7 +354,6 @@ hid_batch_init ()
   batch_hid.set_line_cap          = batch_set_line_cap;
   batch_hid.set_line_width        = batch_set_line_width;
   batch_hid.set_draw_xor          = batch_set_draw_xor;
-  batch_hid.set_draw_faded        = batch_set_draw_faded;
   batch_hid.draw_line             = batch_draw_line;
   batch_hid.draw_arc              = batch_draw_arc;
   batch_hid.draw_rect             = batch_draw_rect;
diff --git a/src/hid/common/extents.c b/src/hid/common/extents.c
index 37ff538..2de34ac 100644
--- a/src/hid/common/extents.c
+++ b/src/hid/common/extents.c
@@ -100,11 +100,6 @@ extents_set_draw_xor (hidGC gc, int xor_)
 {
 }
 
-static void
-extents_set_draw_faded (hidGC gc, int faded)
-{
-}
-
 #define PEX(x,w) if (box.X1 > (x)-(w)) box.X1 = (x)-(w); \
 	if (box.X2 < (x)+(w)) box.X2 = (x)+(w)
 #define PEY(y,w) if (box.Y1 > (y)-(w)) box.Y1 = (y)-(w); \
@@ -191,7 +186,6 @@ hid_extents_init (void)
   extents_hid.set_line_cap        = extents_set_line_cap;
   extents_hid.set_line_width      = extents_set_line_width;
   extents_hid.set_draw_xor        = extents_set_draw_xor;
-  extents_hid.set_draw_faded      = extents_set_draw_faded;
   extents_hid.draw_line           = extents_draw_line;
   extents_hid.draw_arc            = extents_draw_arc;
   extents_hid.draw_rect           = extents_draw_rect;
diff --git a/src/hid/gerber/gerber.c b/src/hid/gerber/gerber.c
index 29eeafa..b6d7585 100644
--- a/src/hid/gerber/gerber.c
+++ b/src/hid/gerber/gerber.c
@@ -55,7 +55,6 @@ static void gerber_set_color (hidGC gc, const char *name);
 static void gerber_set_line_cap (hidGC gc, EndCapStyle style);
 static void gerber_set_line_width (hidGC gc, int width);
 static void gerber_set_draw_xor (hidGC gc, int _xor);
-static void gerber_set_draw_faded (hidGC gc, int faded);
 static void gerber_draw_line (hidGC gc, int x1, int y1, int x2, int y2);
 static void gerber_draw_arc (hidGC gc, int cx, int cy, int width, int height, int start_angle, int delta_angle);
 static void gerber_draw_rect (hidGC gc, int x1, int y1, int x2, int y2);
@@ -802,12 +801,6 @@ gerber_set_draw_xor (hidGC gc, int xor_)
 }
 
 static void
-gerber_set_draw_faded (hidGC gc, int faded)
-{
-  ;
-}
-
-static void
 use_gc (hidGC gc, int radius)
 {
   int c;
@@ -1186,7 +1179,6 @@ hid_gerber_init ()
   gerber_hid.set_line_cap        = gerber_set_line_cap;
   gerber_hid.set_line_width      = gerber_set_line_width;
   gerber_hid.set_draw_xor        = gerber_set_draw_xor;
-  gerber_hid.set_draw_faded      = gerber_set_draw_faded;
   gerber_hid.draw_line           = gerber_draw_line;
   gerber_hid.draw_arc            = gerber_draw_arc;
   gerber_hid.draw_rect           = gerber_draw_rect;
diff --git a/src/hid/gtk/gtkhid-gdk.c b/src/hid/gtk/gtkhid-gdk.c
index 8c3af43..27d62c6 100644
--- a/src/hid/gtk/gtkhid-gdk.c
+++ b/src/hid/gtk/gtkhid-gdk.c
@@ -475,12 +475,6 @@ ghid_set_draw_xor (hidGC gc, int xor_mask)
   ghid_set_color (gc, gc->colorname);
 }
 
-void
-ghid_set_draw_faded (hidGC gc, int faded)
-{
-  printf ("ghid_set_draw_faded(%p,%d) -- not implemented\n", gc, faded);
-}
-
 static int
 use_gc (hidGC gc)
 {
diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index 8d5e3d2..f5e2717 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -2123,7 +2123,6 @@ hid_gtk_init ()
   ghid_hid.set_line_cap             = ghid_set_line_cap;
   ghid_hid.set_line_width           = ghid_set_line_width;
   ghid_hid.set_draw_xor             = ghid_set_draw_xor;
-  ghid_hid.set_draw_faded           = ghid_set_draw_faded;
   ghid_hid.draw_line                = ghid_draw_line;
   ghid_hid.draw_arc                 = ghid_draw_arc;
   ghid_hid.draw_rect                = ghid_draw_rect;
diff --git a/src/hid/gtk/gui.h b/src/hid/gtk/gui.h
index 1d5757f..6cdeff2 100644
--- a/src/hid/gtk/gui.h
+++ b/src/hid/gtk/gui.h
@@ -479,7 +479,6 @@ void ghid_set_color (hidGC gc, const char *name);
 void ghid_set_line_cap (hidGC gc, EndCapStyle style);
 void ghid_set_line_width (hidGC gc, int width);
 void ghid_set_draw_xor (hidGC gc, int _xor);
-void ghid_set_draw_faded (hidGC gc, int faded);
 void ghid_draw_line (hidGC gc, int x1, int y1, int x2, int y2);
 void ghid_draw_arc (hidGC gc, int cx, int cy, int xradius, int yradius,
                     int start_angle, int delta_angle);
diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index 554bbae..6a42706 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -3237,12 +3237,6 @@ lesstif_set_draw_xor (hidGC gc, int xor_set)
   gc->xor_set = xor_set;
 }
 
-static void
-lesstif_set_draw_faded (hidGC gc, int faded)
-{
-  /* We don't use this */
-}
-
 #define ISORT(a,b) if (a>b) { a^=b; b^=a; a^=b; }
 
 static void
@@ -3894,7 +3888,6 @@ hid_lesstif_init ()
   lesstif_hid.set_line_cap            = lesstif_set_line_cap;
   lesstif_hid.set_line_width          = lesstif_set_line_width;
   lesstif_hid.set_draw_xor            = lesstif_set_draw_xor;
-  lesstif_hid.set_draw_faded          = lesstif_set_draw_faded;
   lesstif_hid.draw_line               = lesstif_draw_line;
   lesstif_hid.draw_arc                = lesstif_draw_arc;
   lesstif_hid.draw_rect               = lesstif_draw_rect;
diff --git a/src/hid/png/png.c b/src/hid/png/png.c
index e47c641..26f15b7 100644
--- a/src/hid/png/png.c
+++ b/src/hid/png/png.c
@@ -96,7 +96,6 @@ typedef struct hid_gc_struct
   EndCapStyle cap;
   int width;
   unsigned char r, g, b;
-  int faded;
   color_struct *color;
   gdImagePtr brush;
   int is_erase;
@@ -1177,12 +1176,6 @@ png_set_draw_xor (hidGC gc, int xor_)
 }
 
 static void
-png_set_draw_faded (hidGC gc, int faded)
-{
-  gc->faded = faded;
-}
-
-static void
 use_gc (hidGC gc)
 {
   int need_brush = 0;
@@ -1542,7 +1535,6 @@ hid_png_init ()
   png_hid.set_line_cap        = png_set_line_cap;
   png_hid.set_line_width      = png_set_line_width;
   png_hid.set_draw_xor        = png_set_draw_xor;
-  png_hid.set_draw_faded      = png_set_draw_faded;
   png_hid.draw_line           = png_draw_line;
   png_hid.draw_arc            = png_draw_arc;
   png_hid.draw_rect           = png_draw_rect;
diff --git a/src/hid/ps/eps.c b/src/hid/ps/eps.c
index 0aca27e..e17f8a3 100644
--- a/src/hid/ps/eps.c
+++ b/src/hid/ps/eps.c
@@ -42,7 +42,6 @@ static void eps_set_color (hidGC gc, const char *name);
 static void eps_set_line_cap (hidGC gc, EndCapStyle style);
 static void eps_set_line_width (hidGC gc, int width);
 static void eps_set_draw_xor (hidGC gc, int _xor);
-static void eps_set_draw_faded (hidGC gc, int faded);
 static void eps_draw_rect (hidGC gc, int x1, int y1, int x2, int y2);
 static void eps_draw_line (hidGC gc, int x1, int y1, int x2, int y2);
 static void eps_draw_arc (hidGC gc, int cx, int cy, int width, int height, int start_angle, int delta_angle);
@@ -486,12 +485,6 @@ eps_set_draw_xor (hidGC gc, int xor_)
 }
 
 static void
-eps_set_draw_faded (hidGC gc, int faded)
-{
-  ;
-}
-
-static void
 use_gc (hidGC gc)
 {
   if (linewidth != gc->width)
@@ -654,7 +647,6 @@ hid_eps_init ()
   eps_hid.set_line_cap        = eps_set_line_cap;
   eps_hid.set_line_width      = eps_set_line_width;
   eps_hid.set_draw_xor        = eps_set_draw_xor;
-  eps_hid.set_draw_faded      = eps_set_draw_faded;
   eps_hid.draw_line           = eps_draw_line;
   eps_hid.draw_arc            = eps_draw_arc;
   eps_hid.draw_rect           = eps_draw_rect;
